bullish.js

```diff
@@ -6,32 +6,44 @@ const Queue = require('bull');
 
 // register options schema
 const pluginOptionsSchema = joi.object({
-  redis: joi.object({
-    host: joi.string().default('localhost'),
-    port: joi.number().default(6379),
-  }).default(),
-  routes: joi.object({
-    basePath: joi.string().default('/jobs'),
-    tags: joi.array().items(joi.string()).default(['api']),
-  }).default(),
-  isWorker: joi.boolean().default(true),
+  redis: joi
+    .object({
+      host: joi.string().default('localhost'),
+      port: joi.number().default(6379)
+    })
+    .default(),
+  routes: joi
+    .object({
+      basePath: joi.string().default('/jobs'),
+      tags: joi
+        .array()
+        .items(joi.string())
+        .default(['api'])
+    })
+    .default(),
+  isWorker: joi.boolean().default(true)
 });
 
 // bullish.job options schema
 const jobOptionsSchema = joi.object({
   name: joi.string().required(),
   // pipeline: joi.array().items(joi.func()).min(1).required(), // maybe soon
   handler: joi.func().required(),
-  config: joi.object({
-    routes: joi.alternatives().try(
-      joi.boolean().only(false),
-      joi.array().allow(['create', 'status', 'simulate', 'list'])
-    ).default(['create', 'status', 'list']),
-    stripData: joi.boolean().default(false),
-    concurrency: joi.number().positive().default(1),
-    pre: joi.array().min(1),
-    validate: joi.object(),
-  }).default(),
+  config: joi
+    .object({
+      routes: joi
+        .alternatives()
+        .try(joi.boolean().only(false), joi.array().allow(['create', 'status', 'simulate', 'list']))
+        .default(['create', 'status', 'list']),
+      stripData: joi.boolean().default(false),
+      concurrency: joi
+        .number()
+        .positive()
+        .default(1),
+      pre: joi.array().min(1),
+      validate: joi.object()
+    })
+    .default()
```

Comment:
Thanks a lot for all these reformatting stuff. Helps a lot in terms of readability, though it makes it a bit difficult to review this PR. But well, better difficult than ugly :)

Reply:
I had prettier on while saving. I think its a good default. I apologize for making review harder. Sorry

Reply:
IMO it's a good idea to disable any auto formatting tools while working on foreign open source software. In this case it is acceptable :+1:. But imagine a bigger file where you intentionally change 1 line but suddenly have 500 lines changed.